lib/ansible/modules/windows/win_domain_user.ps1

```diff
@@ -185,6 +185,8 @@ If ($state -eq 'present') {
             } else {
                 $add_attributes.$attribute_name = $attribute_value
             }
+        } else {
```

Comment:
I don't think this is right, I was not aware of `foreach` having an `else` statement in .NET.

Reply:
rebase fun - fixed

Reply:
aren't they always :)